p5/pmath/curves.py

```diff
@@ -21,9 +21,12 @@
 
 from collections.abc import Iterable
 from functools import wraps
+from typing import Tuple, Union, Callable
 from ..pmath import Point
 from ..core import p5
 
+FloatOrNTuple = Union[float, Tuple[float, ...]]
```

Comment:
Instead of defining `FloatOrNTuple` [twice](https://github.com/p5py/p5/pull/428/files#diff-19fec8cd0e782d532dba8aff6d7171784df40a9323ffca9c0c81f84a7a3d6fe8R16). Can we make a types helper/util file which would hold all the custom types. High chances of these types being re-used multiple times.

Reply:
then I put them into p5/p5types.py